composer.json

```diff
@@ -4,6 +4,9 @@
     "type": "library",
     "license": "MIT",
     "config": {
+        "platform": {
+            "php": "7.4.7"
+        },
```

Comment:
Adding a platform here is not correct here

Reply:
Ah yes, should be removed :+1: (Was added so that I was able to install the package using my host PHP 7.4.3 interpreter, which is the version that is coupled to Ubuntu)